pulseapi/profiles/views.py

```diff
@@ -13,6 +13,14 @@
 )
 
 
+# utility function for removing keyed values, with key, from a dict
+def remove_key(container, property):
```

Comment:
we don't need this here anymore?

Reply:
curious, normally flake8 catches that. removing.

Reply:
this won't be something that flake8 would catch because this function is essentially an exported function, so it might be used in some other module and flake8 can't detect that.